modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryTableStorage.java

```diff
@@ -438,6 +442,52 @@ private IndexMetaTree createIndexMetaTree(
         }
     }
 
+    /**
+     * Returns new {@link BlobFreeList} instance for partition.
+     *
+     * @param tableView Table configuration.
+     * @param partId Partition ID.
+     * @param pageMemory Persistent page memory instance.
+     * @param meta Partition metadata.
+     * @throws StorageException If failed.
+     */
+    private BlobFreeList createBlobFreeList(
```

Comment:
Technically, we can use `RowVersionFreeList` without creating an additional `FreeList`.